Packages/com.unity.inputsystem/InputSystem/Editor/AssetEditor/InputActionEditorWindow.cs

```diff
@@ -827,7 +827,7 @@ public static AssetDeleteResult OnWillDeleteAsset(string path, RemoveAssetOption
 
             // Handle .inputactions asset being moved.
             // ReSharper disable once UnusedMember.Local
-            public static AssetMoveResult OnWillMoveAsset(string sourcePath, string destinationPath)
+            public static AssetMoveResult OnWillMoveAsset(string sourcePath, string _)
```

Comment:
Need to suppress the warning here for the analyzer to be happy.

Reply:
(the breakage is from users now seeing a compiler warning pop up in the console when they import the input system package)